RoadMap_SwiftUI/RoadMap_SwiftUI/View/PaymentView.swift

```diff
@@ -0,0 +1,391 @@
+        }
+        .padding(.leading, 30)
+    }
+
+    private var validThruDatePickerView: some View {
+        Picker("", selection: $viewModel.validThruDate) {
+            ForEach(viewModel.months, id: \.self) { month in
+                Text(month)
+            }
+        }
+        .pickerStyle(.menu)
+    }
+
+    private var validThruYearPickerView: some View {
+        Picker("", selection: $viewModel.validThruYear) {
+            ForEach(viewModel.years, id: \.self) { year in
+                Text(year)
+            }
+        }
+        .pickerStyle(.menu)
+    }
+
+    private var cardholderTextFieldView: some View {
+        TextField(
+            Constants.Texts.cardholderNameText,
+            text: $viewModel.cardholderName
+        )
+        .focused($cardholderIsFocused)
+        .onSubmit {
+            cardNumberIsFocused = true
+        }
+    }
+
+    private var CVCTextFieldView: some View {
+        SecureField(
+            Constants.Texts.CVCplaceholder,
+            text: $viewModel.CVC
+        )
+        .focused($CVCIsFocused)
+        .onChange(of: viewModel.CVC) { newValue in
+            if viewModel.checkNumberCount(newValue) {
+                CVCIsFocused = false
+            }
+            viewModel.checkCVC(newValue)
+        }
+        .onTapGesture {
+            viewModel.flipFlashcard()
+        }
+        .keyboardType(.numberPad)
+    }
+
+    private var cardNumberTextFieldView: some View {
+        TextField(
+            Constants.Texts.cardNumberPlaceholder,
+            text: $viewModel.cardNumber
+        )
+        .focused($cardNumberIsFocused)
+        .onChange(
+            of: viewModel.cardNumber,
+            perform: { newValue in
+                if viewModel.checkNumberCount(newValue) {
+                    CVCIsFocused = true
+                }
+                viewModel.getNumber(newValue)
+            }
+        )
+        .keyboardType(.numberPad)
+    }
+
+    private var cardFrontSideView: some View {
+        ZStack {
+            cardRoundedRectangeView
+            cardContentStackView
+        }
+    }
+
+    private var cardBackSideView: some View {
+        ZStack {
+            cardRoundedRectangeView
+            cardBackContentStackView
+        }
+    }
+
+    private var cardRoundedRectangeView: some View {
+        RoundedRectangle(cornerRadius: Constants.Numbers.rectangleCornerRadius)
+            .fill(LinearGradient(
+                colors: [Color.yellow, Color.red],
+                startPoint: .topLeading,
+                endPoint: .bottomTrailing
+            ))
+            .frame(width: 320, height: 190)
+            .padding()
+    }
+
+    private var cardBackContentStackView: some View {
+        VStack(alignment: .leading, spacing: 15) {
+            Rectangle()
+                .fill(Color.black)
+                .frame(width: 320, height: 35)
+            CVCStackView
+        }
+    }
+
+    private var CVCStackView: some View {
```

Comment:
```suggestion
    private var cvcStackView: some View {
```
 что это вдруг у нас переменные стали с большой буквы именоваться)